pinot-core/src/test/java/org/apache/pinot/core/operator/combine/SelectionCombineOperatorTest.java

```diff
@@ -84,9 +96,37 @@ public void setUp()
       throws Exception {
     FileUtils.deleteDirectory(TEMP_DIR);
     _indexSegments = new ArrayList<>(NUM_SEGMENTS);
-    for (int i = 0; i < NUM_SEGMENTS; i++) {
+    for (int i = 0; i < NUM_SEGMENTS / 2; i++) {
```

Comment:
Can we rename the below `createSegment()` method to sth like `createOfflineSegment()` since you've added a new `createRealtimeSegment()` method here?